components/StyledInputTags.js

```diff
@@ -1,47 +1,273 @@
+                  ...style,
+                  zIndex: 9999,
+                }}
+              >
+                <StyledCard m={1} overflow="auto" {...props} ref={scrollerRef} boxShadow="0px 4px 10px #C4C7CC">
+                  <InputWrapper color="black.400">
+                    <TagIcon size="16px" />
+                    <Input
+                      data-cy="styled-input-tags-input"
+                      placeholder={formatMessage(messages.placeholder)}
+                      onKeyPress={e => {
+                        handleNewValue(e);
+                      }}
+                      ref={inputRef}
+                    />
+                  </InputWrapper>
+                  {(suggestedTags?.length || tags?.length) > 0 && (
+                    <Box flexGrow="1">
+                      {suggestedTags
+                        ?.filter(st => !tags.some(t => t.value === st))
+                        .map(st => (
+                          <TagWrapper key={st} px="16px" py="8px" backgroundColor="blue.50">
+                            <StyledTag type="info" variant="rounded-right">
+                              {st}
+                            </StyledTag>
+                            <AddTagButton
+                              data-cy={`styled-input-tags-add-suggestion-${st}`}
+                              onClick={() => {
+                                addTag(st);
+                              }}
+                            >
+                              <Plus size="10px" />
+                            </AddTagButton>
+                          </TagWrapper>
+                        ))}
+                      {tags.map(tag => (
+                        <TagWrapper key={tag.value} px="16px" py="8px" autoFocus>
+                          <StyledTag variant="rounded-right">{tag.label}</StyledTag>
+                          <DeleteTagButton
+                            data-cy={`styled-input-tags-remove-${tag.value}`}
+                            onClick={() => {
+                              removeTag(tag.value);
+                            }}
+                          >
+                            <Times size="10px" />
+                          </DeleteTagButton>
+                        </TagWrapper>
+                      ))}
+                    </Box>
+                  )}
+                </StyledCard>
+              </div>
+            )}
+          </Popper>
+        )}
+      </Flex>
+    </Manager>
   );
 };
 
 StyledInputTags.propTypes = {
-  /** Id of the input */
-  inputId: PropTypes.string,
   suggestedTags: PropTypes.arrayOf(PropTypes.string),
   defaultValue: PropTypes.arrayOf(PropTypes.string),
   value: PropTypes.arrayOf(PropTypes.string),
+  /** True if tags should be rendered despite the value (while still being added/edited) */
+  renderUpdatedTags: PropTypes.bool,
+  onChange: PropTypes.func,
+  ...StyledCard.propTypes,
+};
+
+StyledInputTags.defaultProps = {
+  maxHeight: '30vh',
```

Comment:
**[:sparkles: **Enhancement**]** Not sure that this will plays nicely with smartphones - especially in landscape mode. But we can follow up on accessibility in a separate PR.

---

<details>
<summary>More details about this feedback</summary>
<br/>

This feedback categorization is inspired by [Netlify's Feedback Ladders system](https://www.netlify.com/blog/2020/03/05/feedback-ladders-how-we-encode-code-reviews-at-netlify). It includes 5 levels of feedback:

:rotating_light: **Critical blocker**
This feedback blocks all related work, and requires immediate action. 

:no_entry_sign: **Blocker**
This feedback should block the work from moving forward or being approved, but it does not necessarily require immediate action from the team.

:hammer_and_wrench: **Important**
This feedback should not block the work from moving forward or being approved, but it does require future action from the team. This generally means the feedback is not required for the MVP or initial draft of this work, but should be addressed at some point and we should ideally create an issue to track it.

:sparkles: **Enhancement**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they agree with it.

:ice_cream: **Details & personal preferences**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they deem it useful. This is also the category for everything that relates to personal preferences or code style.

</details>

Reply:
I adjusted this a little bit, it felt it could be a bit bigger in the smartphone.